sdk/search/search-documents/test/utils/setupIndex.ts

```diff
@@ -4,6 +4,10 @@
 import {
   SearchClient,
```

Comment:
I will rename this file to setup.ts at the end in a separate PR. If I do it in this PR, you will not be able to tell what I have added newly.